rlbot_gui/gui/js/botmanager.js

```diff
@@ -0,0 +1,154 @@
+var globalJson
```

Comment:
are the other js files using tabs?

Reply:
tabs are a bit onconsistant in this file, should be fixed next update of the pull